docs/source/publish_and_share_kedro_viz_on_azure.md

```diff
@@ -0,0 +1,128 @@
+# Publish and share via Azure
+
+This page describes how to publish Kedro-Viz to Azure to share it with others. It uses the spaceflights tutorial as an example.
+
+## Setup your kedro project 
+
+If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
+
+```{important}
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.1). To check the version installed, type `kedro -V` in your terminal window.
+```
+
+In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
+
+
+```bash
+kedro new --starter=spaceflights-pandas
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+
+When your project is ready, navigate to the root directory of the project. Install the dependencies from the project root directory by typing the following in your terminal:
+
+```bash
+pip install -r requirements.txt
+```
+
+Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project. Ensure you have these correct versions by updating the `requirements.txt` file of the Kedro project to add the following:
+
+```text
+fsspec[s3]>=2023.9.0
+kedro>=0.18.2
+```
+
+## Install cloud dependencies
+ 
+ Step 1:
+ 
+```bash
+pip install 'kedro-viz[azure]'
+```
+
+## Configure your Azure Blob Storage
+
+You can host your Kedro-Viz project on Azure Blob Storage. 
+
+1. Enable static website hosting - Follow the [tutorial](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website-how-to?tabs=azure-portal) to configure static website hosting on Azure Blob Storage. Please note that uploading your site's files will be done through Kedro-Viz
+
```

Comment:
if we have the below note, we can remove - `Please note that uploading your site's files will be done through Kedro-Viz` statement from above